### PR TITLE
Update forest inventory analysis dataset

### LINKED ISSUE #766: Reinstate Fia script (Forest Inventory and Analysis)
Forest Inventory and Analysis changed the file locations.
New species table url: https://apps.fs.usda.gov/fiadb-downloads/CSV/REF_SPECIES.csv
new main:

The new changes needed for FIA

        self.urls = {"main": "https://apps.fs.usda.gov/fiadb-downloads/CSV/CSV/",
                     'species': 'https://apps.fs.usda.gov/fiadb-downloads/CSV/REF_SPECIES.csv'}

### PR DESCRIPTION
**Pull request by henrykironde**
The location of the files was changed

fixes #766

### CODE REVIEW

**Comment by henrykironde** [comment]
Still waiting for their server. its down currently.

**Comment by ethanwhite** [comment]
Looks to be working for me through the web interface this morning.

File: scripts/fia.py [hunk 1]
```diff
@@ -21,10 +21,10 @@ def __init__(self, **kwargs):
         self.name = "Forest Inventory and Analysis"
         self.shortname = "FIA"
         self.retriever_minimum_version = '2.0.dev'
-        self.version = '1.1'
+        self.version = '1.2'
         self.ref = "http://fia.fs.fed.us/"
-        self.urls = {"main": "http://apps.fs.fed.us/fiadb-downloads/CSV/",
-                     'species': 'http://apps.fs.fed.us/fiadb-downloads/CSV/REF_SPECIES.csv'}
+        self.urls = {"main": "https://apps.fs.usda.gov/fiadb-downloads/CSV/CSV/",
```
Comment (ethanwhite) [inline scripts/fia.py]: Remove extra `CSV`

**Comment by ethanwhite** [comment]
Waiting on #763.

**Pull request merged**